Sources/PackageLoading/PackageBuilder.swift

```diff
@@ -592,7 +598,23 @@ public final class PackageBuilder {
             let path = try findPath(for: target)
             return PotentialModule(name: target.name, path: path, type: target.type)
         })
-        return try createModules(potentialTargets)
+
+        let targets = try createModules(potentialTargets)
+
+        let snippetTargets: [Target]
+
+        if isBuildingRootPackage {
```

Comment:
Could we use `manifest.packageKind == .root` to avoid the new parameter?

Reply:
Ah, if that's enough, certainly!